magda-content-api/src/schemas.ts

```diff
@@ -160,7 +160,6 @@ export const footerCopyright: any = {
     required: ["order", "href", "logoSrc", "htmlContent"]
```

Comment:
Do we need to add schema (for lang setting record) here to make content API accept it?

Reply:
Yes :(